index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body background="./Images for portfolio/header-shapes mobile@2x.svg">
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <a class="logo-text">Sameed Mohsin</a>
+            </div>
+
+            <div class="menu">
+                <img src="./Images for portfolio/Hamburger.svg" alt="" class="hamburger">
+            </div>
+        </nav>
+    </header>
+
+    <section>
+        <div>
+            <h1 class = "top-heading"> I'm Sameed</h1>
+        </div>
+        <p class="my-description">
+            I am a web developer. You can count on me for building, developing, and maintaining your websites.
+        </p>
+        
+        <span class="connect-description">LET'S CONNECT</span>
+        <ul class="connect">
+            <li>
+                <a href="#" class="icon">
+                    <img class="github" src="./Images for portfolio/github.svg" alt="GitHub">
+                </a>
+            </li>
+            <li>
+                <a href="#" class="icon">
+                    <img src="./Images for portfolio/Twitter.svg" alt="Twitter" class="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="#" class="icon">
+                    <img src="./Images for portfolio/Linkedin.svg" alt="LinkedIn" class="linkedin">
+                </a>
+            </li>
+        </ul>
```

Comment:
[OPTIONAL]

- Kindly add all the social icons present on your portfolio design and ensure that the design matches the FIgma design

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212498598-f33e13dc-8e40-4286-acef-be37a11c6e16.png)

FIgma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212498623-0698b512-7dab-4c23-a1ef-ea2aab707204.png)


- [ ]